README.md

```diff
@@ -1,7 +1,7 @@
 <!--
-# SPDX-FileCopyrightText: 2021 Open Networking Foundation <info@opennetworking.org>
-# Copyright 2019 free5GC.org
-
+SPDX-FileCopyrightText: 2021 Open Networking Foundation <info@opennetworking.org>
+SPDX-FileCopyrightText: 2019 free5GC.org
```

Comment:
Please do not change/update the free5gc copyright. Leave it as it was before. My comment about the `SPDX-FileCopyrightText` was applicable only to the Canonical copyright
```suggestion
Copyright 2019 free5GC.org
```